src/manifolds/Stiefel.jl

```diff
@@ -577,7 +566,7 @@ A_{ij}&\text{ if } i > j\\
 -A_{ji} \text{ if } i < j.\\
 \end{cases}
 ```
-[^AbsilMahonySepulchre2008]:
+ [AbsilMahonySepulchre:2008](@cite):
     >Absil, P.-A., Mahony, R. and Sepulchre R.,
     > _Optimization Algorithms on Matrix Manifolds_
```

Comment:
This should be removed

Reply:
Well-spotted, I was looking for ]: but as you can see here I did the cite replace first by accident.